app/next.config.mjs

```diff
@@ -14,6 +14,19 @@ const nextConfig = {
   env: {
     APP_VERSION: packageInfo.version,
   },
+   async headers() {
+    return [
+      {
+        source: "/", 
+        headers: [
+          {
+            key: "Cache-Control",
+            value: "no-store, no-cache, must-revalidate, proxy-revalidate",
```

Comment:
What does this fix? This means we're no longer caching any pages or scripts?

Reply:
This is an extra step meant to prevent pages from being cached I believe otherwise we could remove it if it affects our current caching mechanism . :crossed_fingers: Thanks :+1: